fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -420,6 +513,22 @@ void simpleInsertAndSearch() {
         }
     }
 
+    @Test
+    void largeMetadataTest() {
```

Comment:
Given the behavior of this change, it seems like there should be tests where we have different documents with different sets of fields. @jleach4 Did you find that those tests already existed. 
Perhaps changing this test to use `MAP_DOC` and have a variety of different keys in the map, for each document would cover that.

Reply:
Added tests that use various fields out of the ManyFieldsDoc.